### ISSUE TITLE
[BUG] Slow or failing hostname resolving when using (override) `crc` as External switch after CRC vm stopped on Windows

### ISSUE THREAD

**Issue by jeffsaremi** [code-ready/crc#1078]
### General information

  * OS: Windows
  * Hypervisor: Hyper-V
  * Did you run `crc setup` before starting it (Yes/No)? Yes

## CRC version
```bash
crc version: 1.6.0+8ef676f
OpenShift version: 4.3.0 (embedded in binary)
```
  
## CRC status
```bash
CRC VM:          Stopped
OpenShift:       Stopped
Disk Usage:      0B of 0B (Inside the CRC VM)
Cache Usage:     12.69GB
```

## CRC config
```bash
// nothing
```

## Host Operating System
```bash
Host Name:                 JESAREMI-HP
OS Name:                   Microsoft Windows 10 Enterprise
OS Version:                10.0.18363 N/A Build 18363
OS Manufacturer:           Microsoft Corporation
OS Configuration:          Member Workstation
OS Build Type:             Multiprocessor Free
Product ID:                00330-80000-00000-AA800
Original Install Date:     7/30/2019, 11:45:53 AM
System Boot Time:          2/15/2020, 8:06:58 PM
System Manufacturer:       HP
System Model:              HP Z2 Tower G4 Workstation
System Type:               x64-based PC
Processor(s):              1 Processor(s) Installed.
                           [01]: Intel64 Family 6 Model 158 Stepping 10 GenuineIntel ~3192 Mhz
BIOS Version:              HP Q50 Ver. 01.05.04, 6/17/2019
Windows Directory:         C:\windows
System Directory:          C:\windows\system32
Boot Device:               \Device\HarddiskVolume4
System Locale:             en-us;English (United States)
Input Locale:              en-us;English (United States)
Time Zone:                 (UTC-08:00) Pacific Time (US & Canada)
Total Physical Memory:     65,387 MB
Available Physical Memory: 51,798 MB
Virtual Memory: Max Size:  75,115 MB
Virtual Memory: Available: 55,934 MB
Virtual Memory: In Use:    19,181 MB
Page File Location(s):     C:\pagefile.sys
Hotfix(s):                 16 Hotfix(s) Installed.
                           [01]: KB4534132
                           [02]: KB4493478
                           [03]: KB4497727
                           [04]: KB4503308
                           [05]: KB4508433
                           [06]: KB4509096
                           [07]: KB4515383
                           [08]: KB4516115
                           [09]: KB4517245
                           [10]: KB4520390
                           [11]: KB4521863
                           [12]: KB4524569
                           [13]: KB4528759
                           [14]: KB4537759
                           [15]: KB4538674
                           [16]: KB4532693
Network Card(s):           3 NIC(s) Installed.
                           [01]: Intel(R) Ethernet Connection (7) I219-LM
                                 Connection Name: Ethernet
                                 DHCP Enabled:    Yes
                                 DHCP Server:     N/A
                                 IP address(es)
                           [02]: Hyper-V Virtual Ethernet Adapter
                                 Connection Name: vEthernet (Default Switch)
                                 DHCP Enabled:    No
                                 IP address(es)
                                 [01]: 192.168.207.193
                                 [02]: fe80::811c:dc54:cd74:642e
                           [03]: Hyper-V Virtual Ethernet Adapter
                                 Connection Name: vEthernet (crc)
                                 DHCP Enabled:    Yes
                                 DHCP Server:     10.221.228.18
                                 IP address(es)
                                 [01]: 10.91.90.45
                                 [02]: fe80::549e:1f8c:6cf5:9d10
                                 [03]: 2001:4898:e0:bf:7568:eb8d:c882:df2d
                                 [04]: 2001:4898:e0:bf:549e:1f8c:6cf5:9d10
Hyper-V Requirements:      A hypervisor has been detected. Features required for Hyper-V will not be displayed.
```

### Steps to reproduce

Run
oc stop
Then you cannot access any external sites anymore

### Expected
Internet access to continue working

### Actual
Internet stops working
Networking shows connected
Remote desktop session into the box continues to work

### Logs

https://gist.github.com/jeffsaremi/4ea5862fa6254c8c4e76a946730b71ce

**Comment by gbraad** [comment]
>  Then you cannot access any external sites anymore

You could, but it is the timeout that is an issue...

We added a nameserver to the Default Switch to allow CRC's addresses to be resolved. However, it seems Windows does not handle unreachable state well when the CRC vm is stopped. You can remove the nameserver from the Default Switch. Use `Get-DNSClientServerAddress` to get this information.

**Comment by gbraad** [comment]
> Connection Name: vEthernet (crc)

Perhaps the issue is worse since you use the override switch `crc` that is actually ONLY meant for people on Enterprise that might not have a Default Switch... or for advanced use to define an 'external virtual switch'. Is that the case here?

**Comment by jeffsaremi** [comment]
@gbraad 
Thanks for the hints 
I will apply those
I will also look for a solution without the use of "crc" with the external card. This external switch seems to be disrupting things too much. I'll explore a proxy option.

**Comment by gbraad** [comment]
> I will also look for a solution without the use of "crc" with the external card. This external switch seems to be disrupting things too much. I'll explore a proxy option.

Please keep us to update.

Did you define CRC as an external switch to allow remote access to the OpenShift environment? Need to kind of understand what is happening.

**Comment by gbraad** [comment]
```
                                 Connection Name: vEthernet (crc)
                                 DHCP Server:     10.221.228.18
                                 [01]: 10.91.90.45
```

Likely connected to an office network (or perhaps even VPN?). So I guess what happens with the DNS we add is that this gets routed differently due to the overlap of the network segment with the normally expected work connection. Just guessing here ...

**Comment by jeffsaremi** [comment]
Yes. My crc switch in Hyper-v is External. I can then use `crc ip` directly from outside this machine

**Comment by gbraad** [comment]
Duplicate of #926
